zkevm-circuits/src/witness/block.rs

```diff
@@ -138,7 +136,8 @@ impl<F: Field> Block<F> {
             self.copy_events.iter().map(|c| c.bytes.len() * 2).sum();
         let num_rows_required_for_keccak_table: usize = self.keccak_inputs.len();
         let num_rows_required_for_tx_table: usize =
-            TX_LEN * self.circuits_params.max_txs + self.circuits_params.max_calldata;
+            // FIXME: change it to TX_LEN
+            20 * self.circuits_params.max_txs + self.circuits_params.max_calldata;
         let num_rows_required_for_exp_table: usize = self
```

Comment:
need to fix?

Reply:
I will fix it in the tx circuit PR as this const var `TX_LEN` is defined in `tx_circuit.rs` and I have comment the tx_circuit out in the `lib.rs`